pom.xml

```diff
@@ -14,7 +14,7 @@
 
   <name>Logstash</name>
   <description>A Logstash agent to send Jenkins logs to a Logstash indexer.</description>
-  <url>https://wiki.jenkins.io/display/JENKINS/Logstash+Plugin</url>
+  <url>https://github.com/jenkinsci/logstash-plugin</url>
```

Comment:
this is the line that means pull docs from github,

a release is required to change the plugin site to use github, after the one-off release you won't need to release to update documentation